contrib/cirrus/lib.sh

```diff
@@ -127,6 +133,10 @@ die() {
     exit ${1:-1}
 }
 
+warn() {
+    echo ">>>>> WARNING: ${1:-WARNING (but no message given!) in ${FUNCNAME[1]}()}" > /dev/stderr
```

Comment:
I see `warn` used a lot for merely informational messages. The big `WARNING` here might mislead people into thinking something is wrong. One option might be to remove the 'WARNING' from here; another (which I lean slightly more towards) would be to limit the use of `warn` to actual warnings, and use plain 'echo` for informationals.

Reply:
Yes this was true, though I just grepped through 'contrib/cirrus/' and don't see any that shouldn't be warnings.